package.json

```diff
@@ -35,7 +35,7 @@
     "uglify-js": "^3.4.9"
   },
   "dependencies": {
-    "@mapbox/mapbox-gl-sync-move": "^0.2.0"
+    "@mapbox/mapbox-gl-sync-move": "^0.3.0"
```

Comment:
Is the version change needed?

Reply:
I think they fixed some things in `0.3.0` that we need to behave correctly.

Reply:
That new version exposes a method to clear the sync.